src/js/jsx/sections/transform/Position.jsx

```diff
@@ -99,6 +83,10 @@ define(function (require, exports, module) {
             });
         },
 
+        componentWillMount: function () {
+            this.componentWillReceiveProps(this.props);
```

Comment:
Can you add a comment explaining why you need to call this life-cycle function here? I'm sure it'll confuse someone later.